lib/credentials/shared_ini_file_credentials.js

```diff
@@ -48,11 +51,11 @@ AWS.SharedIniFileCredentials = AWS.util.inherit(AWS.Credentials, {
 
     this.filename = options.filename;
     this.profile = options.profile || process.env.AWS_PROFILE || 'default';
-    this.get(function() {});
```

Comment:
Nevermind, after further testing, turns out this is not an issue since we didn't remove the `SharedIniFileCredentials` from the initial `CredentialProviderChain`.